src/libs/actions/Welcome.ts

```diff
@@ -58,6 +52,30 @@ Onyx.connect({
     },
 });
 
+Onyx.connect({
+    key: ONYXKEYS.NVP_INTRO_SELECTED,
+    initWithStoredValues: true,
+    callback: (value) => {
+        // If isFirstTimeNewExpensifyUser was true do not update it to false. We update it to false inside the Welcome.show logic
```

Comment:
Should this say ` hasSelectedChoice` not `isFirstTimeNewExpensifyUser`? The linked discussion seems unrelated.